### PR TITLE
chore: update capi-utils

### PR DESCRIPTION
**Pull request by Unix4ever**
This update should address some compatibility issues with the latest
cluster API components.

Signed-off-by: Artem Chernyshev <artem.chernyshev@talos-systems.com>

### CODE REVIEW

**Review by frezbo — DISMISSED** [review]

**Comment by smira** [comment]
there's no update to `capi-utils` here?

**Comment by Unix4ever** [comment]
> there's no update to capi-utils here? 

Yeah... oops. Updated

File: go.mod [hunk 1]
```diff
@@ -20,114 +20,195 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.3.0
 	github.com/stretchr/testify v1.7.0
-	github.com/talos-systems/capi-utils v0.0.0-20211126110629-e8c3bf93e75f
+	github.com/talos-systems/capi-utils v0.0.0-20220317162331-3a6f8ee7b753
 	github.com/talos-systems/go-retry v0.3.1
 	github.com/talos-systems/grpc-proxy v0.2.0
 	github.com/talos-systems/talos v0.15.0-alpha.1.0.20220127120728-831f65a07f3b
```
Comment (smira) [inline go.mod:26]: should we upgrade Talos as well? use latest beta?

Comment (Unix4ever) [reply go.mod:26]: sure, let's do it.

**Review by smira — DISMISSED** [review]

**Review by frezbo — APPROVED** [review]

**Pull request merged**